internal/gcsx/bucket_manager_test.go

```diff
@@ -0,0 +1,129 @@
+package gcsx
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/fsouza/fake-gcs-server/fakestorage"
+	"github.com/googlecloudplatform/gcsfuse/internal/storage"
+	"github.com/jacobsa/gcloud/gcs"
+	"github.com/jacobsa/gcloud/gcs/gcsfake"
+	. "github.com/jacobsa/ogletest"
+	"github.com/jacobsa/timeutil"
+)
+
+func TestFlags(t *testing.T) { RunTests(t) }
+
+const TestBucketName string = "gcsfuse-default-bucket"
+
+////////////////////////////////////////////////////////////////////////
+// Boilerplate
+////////////////////////////////////////////////////////////////////////
+
+type BucketManagerTest struct {
+	fakeStorageServer *fakestorage.Server
+	bucketHandle      *storage.BucketHandle
+}
+
+var _ SetUpInterface = &BucketManagerTest{}
+var _ TearDownInterface = &BucketManagerTest{}
+
+func init() { RegisterTestSuite(&BucketManagerTest{}) }
+
+func (t *BucketManagerTest) SetUp(_ *TestInfo) {
+	var err error
+	t.fakeStorageServer, err = storage.CreateFakeStorageServer([]fakestorage.Object{storage.GetTestFakeStorageObject()})
+	AssertEq(nil, err)
+
+	storageClient := &storage.Storageclient{Client: t.fakeStorageServer.Client()}
+	t.bucketHandle, err = storageClient.BucketHandle(TestBucketName)
+	AssertEq(nil, err)
+	AssertNe(nil, t.bucketHandle)
+}
+
+func (t *BucketManagerTest) TearDown() {
+	t.fakeStorageServer.Stop()
+}
+
+func (t *BucketManagerTest) TestNewBucketManagerMethod() {
+	var nilValue *bucketManager = nil
+	storageClient := &storage.Storageclient{Client: t.fakeStorageServer.Client()}
+
```

Comment:
new lines should be between
setup
<<new line>>
test method invocation
<<new line>>
assert statement

Reply:
fixed